src/m365/pp/commands/copilot/copilot-get.ts

```diff
@@ -89,21 +93,21 @@ class PpChatbotGetCommand extends PowerPlatformCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     if (this.verbose) {
```

Comment:
As first thing to do, let's log a deprecation message to notify our users that the old name will go away. You can use the `this.showDeprecationWarning(...)` for this.